docs/edgeql/admin/roles.rst

```diff
@@ -19,20 +19,24 @@ Create a role.
 
 .. eql:synopsis::
 
-    CREATE ROLE <name> [ EXTENDING <base> [, ...] ]
+    CREATE [ SUPERUSER ] ROLE <name> [ EXTENDING <base> [, ...] ]
     "{" <subcommand>; [...] "}" ;
 
     # where <subcommand> is one of
 
-      SET allow_login := {true | false}
-      SET is_superuser := {true | false}
       SET password := <password>
 
 
 Description
 -----------
 
-``CREATE ROLE`` defines a new role.
+``CREATE ROLE`` defines a new database role.
+
+:eql:synopsis:`SUPERUSER`
+    If specified, the created role will have the *superuser* status, and
+    will be exempt from all permission checks.  Currently,
+    the ``SUPERUSER`` qualifier is mandatory, i.e. it is not possible to
+    create non-superuser roles for now.
```

Comment:
Also add an entry to `changelog/1_0_a2.rst` :)

Reply:
I will on my editorial pass.